ow_materials/src/MaterialDistributionPlugin.cpp

```diff
@@ -345,7 +342,7 @@ void MaterialDistributionPlugin::handleVisualBulk(MaterialBlend const &blend)
        "the bulk contains\n";
   for (auto const &x : blend.getBlendMap())
     s << "\t" << x.second << "%% of " << static_cast<int>(x.first) << "\n";
-  gzlog << s.str() << endl;
+  // gzlog << s.str() << endl;
```

Comment:
Is this supposed to be commented out? Will `s` be used for anything at all now?

Reply:
They are just debugging messages. they print 5 lines per modification that look like this
```
(1700596155 692425365) handleCollisionBulk: STUBBED
the bulk contains
        69.5007% of 1
        22.7597% of 2
        7.73968% of 0
```
There are a huge number of modifications per second, so they clutter up the gazebo log quite a bit and drown out the batch statistics log. I will add a comment, but would rather leave both disabled for now.